bitrise.yml

```diff
@@ -0,0 +1,155 @@
+---
+format_version: '8'
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+project_type: ios
+trigger_map:
+- push_branch: dev
+  workflow: deploy_staging
+- push_branch: master
+  workflow: deploy_appstore
+- push_branch: "*"
+  workflow: test
+workflows:
+  deploy_staging:
+    steps:
+    - activate-ssh-key@4:
+        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
+    - git-clone@6.0: {}
```

Comment:
```
- script@1:
        inputs:
        - content: |-
            #!/bin/bash
            set -ex
            RBENV_DIR="`cd $(rbenv which ruby)/../..;pwd`"
            echo "Gem cache directory: $RBENV_DIR"
            envman add --key GEM_CACHE_PATH --value $RBENV_DIR
        title: Set GEM_CACHE_PATH env var
    - git-clone@4.0.17: {}
```

this script sets the ruby gem path so it can be cached in the end